# Some fixed positions that would be goten before takeoff by GPS (GCS->Pix->Mavros)
float32 home_x        #home point X position (NED), in m
float32 home_y        #home point Y position (NED), in m
float32 home_z        #home point Z position (NED), in m
float32 observe_x     #observe point X position (NED), in m
float32 observe_y     #observe point Y position (NED), in m
float32 observe_z     #observe point Z position (NED), in m
float32 spray_left_x  #spray point X position (NED) (left), in m
float32 spray_left_y  #spray point Y position (NED) (left), in m
float32 spray_left_z  #spray point Z position (NED) (left), in m
float32 spray_right_x #spray point X position (NED) (right), in m
float32 spray_right_y #spray point Y position (NED) (right), in m
float32 spray_right_z #spray point Z position (NED) (right), in m
